sidecar/src/interface.rs

```diff
@@ -480,7 +480,10 @@ impl EnqueuedTelemetryData {
                     }
                     async fn parse(path: &PathBuf) -> Result<Vec<data::Dependency>> {
                         let mut json = tokio::fs::read(&path).await?;
+                        #[cfg(not(target_arch = "x86"))]
                         let parsed: ComposerPackages = simd_json::from_slice(json.as_mut_slice())?;
+                        #[cfg(target_arch = "x86")]
```

Comment:
Which project is interested in x86?

Reply:
I don't know, the sidecar isn't. Just making it pass here.